ms2deepscore/SpectrumBinner.py

```diff
@@ -120,7 +120,9 @@ def transform(self, input_spectrums: List[SpectrumType],
             assert 100*missing_fractions[i] <= self.allowed_missing_percentage, \
                 f"{100*missing_fractions[i]:.2f} of weighted spectrum is unknown to the model."
             spectrum = BinnedSpectrum(binned_peaks=create_peak_dict(peak_list),
-                                      metadata={"inchikey": input_spectrums[i].get("inchikey")})
+                                      metadata={"inchikey": input_spectrums[i].get("inchikey"),
+                                                "precursor_mz": input_spectrums[i].get("precursor_mz"),
+                                                "parent_mass": input_spectrums[i].get("parent_mass")})
```

Comment:
Better make generic to also allow other metadata fields.

Reply:
Yes, agreed @florian-huber, think about chemical compound class features, substructure features, etc.... - for the MS2Query related analysis, I am fine with running it on a working version/branch, as it would be good to know the results in the (very) near future....

Reply:
@justinjjvanderhooft @niekdejonge the current changes should make this very flexible and allow running old code as before. So I will merge it today (once tests pass) and then make a new release. 
(thanks @djoas)

Reply:
Nice work! 😎 @florian-huber are you planning to run this now including parent masses?